src/components/partySelectionSearch/PartyItem.tsx

```diff
@@ -55,15 +55,13 @@ export default function PartyItem({
   pxTitleSubTitle,
 }: Props) {
   return (
-    <CustomList
-      aria-label="main mailbox folders"
-      sx={{ border: borderList, backgroundColor: bgColor }}
-    >
+    <CustomList sx={{ border: borderList, backgroundColor: bgColor }}>
```

Comment:
Is this CustomList necessary? Can we remove it? We are configuring List of List (the inner list with just a ListItemButton) in this way